src/appmixer/openai/core/createFile/createFile.js

```
@@ -84,17 +87,16 @@
                 },
                 response: err.response ? {
-                    data: err.response.data,
-                    status: err.response.status,
-                    statusText: err.response.statusText,
-                    headers: err.response.headers
+                    message: err.response.data.error.message,
                 } : undefined
             };
-            await context.log(log);
-            throw err;
+            throw new context.CancelError(log);;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will throw an exception beforehand


(lint/correctness/noUnreachable)